src/btree.h

```diff
@@ -869,3 +872,92 @@ void BTree<K, V, CompareFn>::addToNode(
 	current_node->setIsChangedInMem(true);
 
 };
+
+
+template <typename K, typename V, typename CompareFn>
+BTreeNode<K, V, CompareFn>& BTree<K, V, CompareFn>::getNodeFromBlockNum(blocknum_t block_number){
+	
```

Comment:
Im assuming the block_number will never be wring in which case there will never be nullptr so better to use reference. What do you think?